### PR TITLE
feat: explain the way to use the ApiToken

### PR DESCRIPTION
**Pull request by Isammoc**
Motivation:
New customer failed to use the ApiToken

Modifications:
 * Add an explanation for Authorization header and `bearer ` prefix

Result:
Less errors from users?

### CODE REVIEW

File: content/reference/user/api/index.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ IDs can be retrieved on table either under kebab menu, or on the right of the ta
 
 Some information before using the public API:
 
+- ApiToken must be present in the `Authorization` header after the `bearer ` prefix (exactly `bearer ` with one space)
```
Comment (slandelle) [inline content/reference/user/api/index.md:24]: - The `Authorization` header is mandatory. Its value must be `bearer ` concatenated with the value of your ApiToken. Beware of the white space after the `bearer` prefix.

**Review by slandelle — CHANGES_REQUESTED** [review]

File: content/reference/user/api/index.md (hunk 1 above)
Comment (slandelle) [reply content/reference/user/api/index.md:24]: Note: do we really support both `bearer` and `Bearer`???

Header values are case sensitive and `Bearer` is definitely the standard form, see https://www.rfc-editor.org/rfc/rfc6750#section-6.1.1
That's the form (least unexpected) that we should emphasize.

Comment (Isammoc) [reply content/reference/user/api/index.md:25]: we remove the first 8 characters.

Comment (slandelle) [reply content/reference/user/api/index.md:25]: > we remove the first 8 characters.

Instead of validating the format? 😱

**Comment by Isammoc** [comment]
closed, because it is not like that.

No prefix needed at all...

**Comment by slandelle** [comment]
LOL

**Pull request closed without merging**